.github/workflows/acceptance_tests.yaml

```diff
@@ -29,11 +30,9 @@ jobs:
         with:
           go-version: ${{ steps.go-version.outputs.content }}
       - uses: engineerd/setup-kind@v0.5.0
-      - name: test (server side plan enabled)
-        run: |
-          make testacc KUBE_CONFIG_PATH="${HOME}/.kube/config"
-      - name: test (server side plan disabled)
-        env:
-          TESTARGS: "-server-side-plan=false"
+        with:
+          version: "v0.10.0"
+      - name: Run tests
         run: |
-          make testacc KUBE_CONFIG_PATH="${HOME}/.kube/config"
+          make testacc KUBE_CONFIG_PATH="${HOME}/.kube/config" TF_ACC_LOG_PATH=$HOME/tftest.log
```

Comment:
I'd like to add `kubectl version` to this run section, so we can see what API version we're working with. That could help diagnose what's going on with the PSP annotation. What do you think?